docs/life-of-a-request.md

```diff
@@ -1,33 +1,39 @@
 # Life of a Request
 
 It can sometimes be confusing to figure out how to write a new feature,
-or debug an existing one. Let's try to follow a request through the
+or debug an existing one. Let us try to follow a request through the
 Zulip codebase, and dive deep into how each part works.
 
-## A request is sent to the server, and handled by Nginx
+We will use as our example the creation of users through the API, but we
+will also highlight how alternative requests are handled.
 
-Everything in Zulip goes through nginx, when deployed in production.
-For the most part you don't need to know how this works, except for when
+## A request is sent to the server, and handled by [Nginx](http://nginx.org/en/docs/)
```

Comment:
Maybe add a note at the end fo this paragraph that in development, `tools/run-dev.py` fills the role of `nginx`.